docs/source/releases/v1_12_2a0.rst

```diff
@@ -67,6 +67,24 @@ useful. Now, only useful AttributeErrors should be raised.
     modified: geoips/commandline/log_setup.py
     added: tests/unit_tests/commandline/log_setup.py
 
+Remove unnessesary checks for and mentions of imagemagick
```

Comment:
I tested on a fresh machine and got a different result. 2 of the 3 tests from base_test.sh passed, however the test on ``amsr2_ocean.tc.windspeed.imagery_clean.sh`` failed because I was missing the ``pixelmatch`` module. Checking pyproject.toml, I see this listed as an optional dependency. @jsolbrig do you think it would be a good idea to mark this as a required dependency instead?

Reply:
Here's the error output as well:
```
Traceback (most recent call last):
  File "/Users/erose/anaconda3/envs/geoips/bin/run_procflow", line 8, in <module>
    sys.exit(main())
  File "/Users/erose/geoips/geoips_packages/geoips/geoips/commandline/run_procflow.py", line 69, in main
    RETVAL = PROCFLOW(COMMAND_LINE_ARGS["filenames"], COMMAND_LINE_ARGS)
  File "/Users/erose/geoips/geoips_packages/geoips/geoips/plugins/modules/procflows/single_source.py", line 1923, in call
    retval += output_checker(
  File "/Users/erose/geoips/geoips_packages/geoips/geoips/plugins/modules/output_checkers/image.py", line 230, in call
    retval = plugin.compare_outputs(
  File "/Users/erose/geoips/geoips_packages/geoips/geoips/interfaces/module_based/output_checkers.py", line 875, in compare_outputs
    ) = self.perform_comparisons(
  File "/Users/erose/geoips/geoips_packages/geoips/geoips/interfaces/module_based/output_checkers.py", line 800, in perform_comparisons
    goodcomps, badcomps, compare_strings = test_product_func(
  File "/Users/erose/geoips/geoips_packages/geoips/geoips/interfaces/module_based/output_checkers.py", line 991, in test_products
    if self.module.outputs_match(
  File "/Users/erose/geoips/geoips_packages/geoips/geoips/plugins/modules/output_checkers/image.py", line 121, in outputs_match
    from pixelmatch.contrib.PIL import pixelmatch
ModuleNotFoundError: No module named 'pixelmatch'
```

Reply:
No - it should be an optional dependency - you only need it if you are running tests, not to just run geoips.

Installation process just needs to make sure geoips[test] is installed if you want to be able to run tests.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/blob/main/docs/source/starter/installation.rst#5-install-the-geoips-git-repository

^ Looks like the documentation installation instructions include the optional "test" - so I think we should be good.

Reply:
Yeah, this should remain as needed only for tests. If we want to run something without installing the test dependencies, we would run it without the `--comparepath` option set. That said, it might be good to look at the errors that are raised when we call run_procflow with `--comparepath` and we haven't installed the test dependencies.

@evrose54 Could you create a new issue to add a more appropriate error when `--comparepath` is used but the test dependencies aren't found? Have it say something about needing to install the test dependencies using `pip instsall geoips[test]` or `pip install .[test]` depending on how you installed.

Reply:
Yup! See here: https://github.com/NRLMMD-GEOIPS/geoips/issues/451